src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1327,6 +1345,8 @@ public function prep_import_data( $data = [] ) {
 		}
 
 		if ( is_wp_error( $data ) ) {
+			tribe( 'logger' )->log_debug( "Data of the import has errors.", 'EA Prepare Import' );
```

Comment:
:no_entry_sign: String "Data of the import has errors." does not require double quotes; use single quotes instead

<sub>hash: f947e1e4e04651375d133525864b1000</sub>